### PR TITLE
Manual project add

### PR DESCRIPTION
**Pull request by ianmah**
TODO (in future pr): Display existing projects and add edit project functionality?
image

image

### CODE REVIEW

**Comment by ianmah** [comment]
https://pr-74.d1shnh93w9du7n.amplifyapp.com/Livesite/judging

File: pages/Livesite/judging.js [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+        <Input
+          value={data.name}
+          onChange={(e) => handleChange('teamMembersEmails', e)}
+        />
+      </Modal>
+    </Page>
+  );
+};
+
+export const getStaticProps = async () => {
```
Comment (jackyzha0) [inline pages/Livesite/judging.js:150]: this seems unused?

**Review by jackyzha0 — APPROVED** [review]
lgtm :O)

File: pages/Livesite/judging.js (hunk 1 above)
Comment (ianmah) [reply pages/Livesite/judging.js:150]: its part of the build process (It gets the list of hackathons and builds the page with that static list)

Comment (ianmah) [reply pages/Livesite/judging.js:150]: https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation

Comment (jackyzha0) [reply pages/Livesite/judging.js:150]: next is funky; looks good

**Pull request merged**